File: 20220928/모범답안/20220928_07.js/CountPostfixWordNumber.py
def solution(array, s):
    """
    :param array: string[]
    :param s: string
    :return: int
    """

    result = 0

    for postfix in array:
        subPostText = s[len(s) - len(postfix):len(s)]
        if subPostText == postfix:
            result += 1

    return result

File: 20220928/모범답안/20220928_07.js/CountPostfixWordNumber.js
/**
 * @param array {string[]}
 * @param s {string}
 * @returns {number}
 */
function solution(array, s) {
  let result = 0

  for (let i = 0; i < array.length; i++) {
    const postfix = array[i]
    const subPostText = s.substring(s.length - postfix.length, s.length)
    if (subPostText === postfix) {
      result++
    }
  }

  return result
}

export default solution
